packages/suite-desktop/package.json

```diff
@@ -177,6 +175,7 @@
     "dependencies": {
         "@trezor/suite-desktop-api": "1.0.0",
         "@trezor/utils": "*",
+        "@trezor/suite": "1.0.0",
```

Comment:
alphabetical order @trezor/utils should be after @trezor/suite

Reply:
fixed